src/Chainweb/Version.hs

```diff
@@ -511,11 +511,23 @@ chainwebGraphs (TimedConsensus g1 g2) = (8, g2) NE.:| [ (0, g1) ]
 chainwebGraphs (PowConsensus g) = pure (0, g)
 chainwebGraphs (TimedCPM g) = pure (0, g)
 chainwebGraphs (FastTimedCPM g) = pure (0, g)
-chainwebGraphs Testnet04 = pure (0, petersonChainGraph)
-chainwebGraphs Mainnet01 = pure (0, petersonChainGraph)
-chainwebGraphs Development = (50, twentyChainGraph) NE.:| [ (0, petersonChainGraph) ]
+chainwebGraphs Testnet04 =
+    ( to20ChainsTestnet, twentyChainGraph ) NE.:|
+    [ ( 0, petersonChainGraph ) ]
+chainwebGraphs Mainnet01 =
+    ( to20ChainsMainnet, twentyChainGraph ) NE.:|
+    [ ( 0, petersonChainGraph ) ]
+chainwebGraphs Development =
+    ( 50, twentyChainGraph ) NE.:|
+    [ ( 0, petersonChainGraph ) ]
 {-# INLINE chainwebGraphs #-}
 
+to20ChainsMainnet :: BlockHeight
+to20ChainsMainnet = 852_054 -- 2020-08-20 16:00:00
```

Comment:
👍 double checked and got `2020-08-20 16:02:36 UTC`

Reply:
👍 on 2020-08-20.